guacamole/src/main/webapp/translations/en.json

```diff
@@ -531,6 +533,7 @@
         "FIELD_HEADER_RECORDING_PATH" : "Recording path:",
         "FIELD_HEADER_SFTP_DIRECTORY"   : "Default upload directory:",
         "FIELD_HEADER_SFTP_HOSTNAME"    : "Hostname:",
+        "FIELD_HEADER_SFTP_KEEPALIVE"   : "SFTP Keepalive Interval:",
```

Comment:
`FIELD_HEADER_SFTP_KEEPALIVE` will not work, as the parameter has been renamed to `sftp-server-alive-interval`. This will need to be changed to `FIELD_HEADER_SFTP_SERVER_ALIVE_INTERVAL`.